lib/include/openamp/rpmsg_virtio.h

```diff
@@ -140,6 +140,15 @@ rpmsg_virtio_create_virtqueues(struct rpmsg_virtio_device *rvdev,
 					callbacks);
 }
 
+static inline int
+rpmsg_virtio_notify_wait(struct rpmsg_virtio_device *rvdev,
```

Comment:
any need to expose this API to the users?

Reply:
Don't need